### PR TITLE
add challenges

### PR DESCRIPTION
**Pull request by mariajbellido**

### CODE REVIEW

File: staff/mariajesus-bellido/challenges/calculator-app/main.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+
+//Declaración de variable 
+//ResetValue = reset calculadora (C)
+//Transformación del dato una vez recogido (String vacía '') -> valor conseguimos que desaparezca el valor
+
+
+var current = ''
```
Comment (Aiidamontero) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js]: falta ;

File: staff/mariajesus-bellido/challenges/calculator-app/main.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+
+//Declaración de variable 
+//ResetValue = reset calculadora (C)
+//Transformación del dato una vez recogido (String vacía '') -> valor conseguimos que desaparezca el valor
+
+
+var current = ''
+var aux = ''
```
Comment (Aiidamontero) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js]: falta ;

File: staff/mariajesus-bellido/challenges/calculator-app/main.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+//"aux" recogerá el valor para añadirlo a "current"
+// ahora ya vaciaremos / limpiamos el display con '' (empty string)
+
+aux = current
+current = ''
+
+// Botones del 0 al 9  (orden ascendente) 
+buttons [13].onclick = function () { 
+    current = current + '0'
```
Comment (Marta108) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js]: Faltaría un punto y coma creo

File: staff/mariajesus-bellido/challenges/calculator-app/main.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    display.innerText = current
+}
+
+buttons [8].onclick = function () { 
+    current = current + '1'
+    display.innerText = current
+}
```
Comment (Marta108) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js]: También, el punto y coma

File: staff/mariajesus-bellido/challenges/calculator-app/main.js [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    function calculate(a, operator, b) {
+        if (operator === '+')
+            return a + b
+        else if (operator === '-')
+            return a - b
+        else if (operator === '*')
+            return a * b
+        else (operator === '/')
+        return a / b
+    }
```
Comment (Crisappletree) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js:132]: This function in a different file named as "Calculator.js"

File: staff/mariajesus-bellido/challenges/calculator-app/main.js [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+buttons [12].onclick = function() {
+    current = resetValue;
+    display.innerText = resetValue;
+}
+
+
+// Botón igual + función calculate 
+buttons [16].onclick = function () { 
+    var result = calculate(Number(aux), operator, Number(current))
```
Comment (Marta108) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js:119]: I think that at the end  , you have to put a point and a coma

File: staff/mariajesus-bellido/challenges/calculator-app/main.js [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+buttons [2].onclick = function () { 
+    current = current + '9'
+    display.innerText = current
+}
+
+// Botones - Operaciones matemáticas (división, multiplicación, resta y suma) 
+
+
+buttons [3].onclick = function () { 
+    aux = current
```
Comment (Marta108) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js]: Point and coma

File: staff/mariajesus-bellido/challenges/calculator-app/main.js [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+    display.innerText = current 
+}
+
+buttons [1].onclick = function () { 
+    current = current + '8' 
+    display.innerText = current 
+}
+
+buttons [2].onclick = function () { 
+    current = current + '9'
```
Comment (Marta108) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js]: Point and coma after the numbers in brakets...I think but not sure

File: staff/mariajesus-bellido/challenges/calculator-app/main.js (hunk 1 above)
Comment (Marta108) [inline staff/mariajesus-bellido/challenges/calculator-app/main.js]: Point and coma again

File: staff/mariajesus-bellido/challenges/calculator-app/index.html [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+        </div>
+            <div class = "equal">
+                <button> = </button>
+            </div>
+        </div>
+    </div>
+
+    <script src="main.js"></script>
+</body>
+</html>
```
Comment (Marta108) [inline staff/mariajesus-bellido/challenges/calculator-app/index.html:48]: I have not seen anything, to be honest

File: staff/mariajesus-bellido/challenges/calculator-app/main.js (hunk 1 above)
Comment (mariajbellido) [reply staff/mariajesus-bellido/challenges/calculator-app/main.js]: done, thanks!

File: staff/mariajesus-bellido/challenges/calculator-app/main.js (hunk 3 above)
Comment (mariajbellido) [reply staff/mariajesus-bellido/challenges/calculator-app/main.js]: corregido, gracias :-)

File: staff/mariajesus-bellido/challenges/calculator-app/main.js (hunk 6 above)
Comment (mariajbellido) [reply staff/mariajesus-bellido/challenges/calculator-app/main.js:119]: semicolons added, thanks!

File: staff/mariajesus-bellido/challenges/calculator-app/main.js (hunk 8 above)
Comment (mariajbellido) [reply staff/mariajesus-bellido/challenges/calculator-app/main.js]: done, thanks!

File: staff/mariajesus-bellido/challenges/calculator-app/main.js (hunk 5 above)
Comment (mariajbellido) [reply staff/mariajesus-bellido/challenges/calculator-app/main.js:132]: Could you please explain this a little bit further? Not sure what you mean with this, sorry! :-(

**Review by simonbesteiro — APPROVED** [review]
review done

**Pull request merged**